### ISSUE TITLE
Support dgraph >= 0.8.0

### ISSUE THREAD

**Issue by reicheltp** [reicheltp/dgraph-client#5]

**Comment by matth3wga0** [comment]
Is this issue related to `Trying to connect an http1.x server`? I'm running version 0.8.2.

**Comment by reicheltp** [comment]
Yes. And there are several other issues targeting dgraph >= 0.8.0 since they have changed the proto files.

If you want to try this lib you are currently bound to dgraph 0.7.7
